File: 242-valid-anagram/valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
        if(s.length()!=t.length()) return false;
        int[] smap= new int[26];

        for(int i=0; i< s.length(); i++){
            char chs=s.charAt(i);
            char cht=t.charAt(i);
            smap[chs-'a']++;
            smap[cht-'a']--;
        }

        for(int i=0; i<26; i++){
            if(smap[i]!=0) return false;
        }

        return true;

    }
}

File: 242-valid-anagram/valid-anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        sCount=[0]*26

        for ch in s:
            sCount[ord(ch)-ord('a')]+=1
        for ch in t:
            sCount[ord(ch)-ord('a')]-=1
        for num in sCount:
            if num!=0:
                return False
        return True